stack/app.py

```diff
@@ -254,9 +254,13 @@ def __init__(
             user_verification=cognito.UserVerificationConfig(
                 # TODO: this email body can contain HTML tags for a better user experience
                 email_body=(
-                    "Thank you for signing up to the Algorithm Publication Tool.<br>Please "
-                    "verify you account by clicking on {##Verify Email##}.<br>Sincerely,"
-                    "<br>The NASA APT team"
+                    "Thank you for signing up to the Algorithm Publication Tool.<br>"
+                    "Please verify you account by clicking on {##Verify Email##}.<br><br>"
+                    "Your account only needs to be verified once. If your account "
+                    "is already verified, you can return to the "
+                    f"<a href='{frontend_url[:-1] if frontend_url.endswith('/') else frontend_url}/signin'>Algorithm Publication Tool</a> to sign in."
```

Comment:
```suggestion
                    f"<a href='{frontend_url.strip('/')}/signin'>Algorithm Publication Tool</a> to sign in."
```
`str.strip(x:str|char)` strips every leading and trailing instance of the char or substring `x` from the string